config/routes.rb

```diff
@@ -1,5 +1,5 @@
 Rails.application.routes.draw do
-  root to: "gatherings#index"
+  root to: redirect("gatherings")
```

Comment:
Just noticed, what is the purpose of  `redirect` here?

Reply:
Removed this in a new version.